src/fuzzfetch/core.py

```diff
@@ -461,6 +461,12 @@ def resolve_targets(self, targets: Sequence[str]) -> None:
                     f"'{self._platform.system}' is not a supported platform"
                 )
 
+        if "mozharness" in targets_remaining:
+            targets_remaining.remove("mozharness")
+            artifact_path = "/".join(self._artifact_base.split("/")[:-1])
```

Comment:
Since we're `Path`-ifying all the things, `_artifact_base` could be a `PurePosixPath` and then this could just be `self._artifact_base.parent`.

Reply:
Agreed.  I'll do all of the Path changes in a separate PR.